src/Raven.Client/ServerWide/ETL/RavenEtlConfiguration.cs

```diff
@@ -13,12 +13,19 @@ public class RavenEtlConfiguration : EtlConfiguration<RavenConnectionString>
 
         public override string GetDestination()
         {
-            return _destination ?? (_destination = $"{Connection.Database}@{Connection.Url}");
+            return _destination ?? (_destination = $"{Connection.Database}@{string.Join(",",Connection.TopologyDiscoveryUrls)}");
         }
 
         public override bool UsingEncryptedCommunicationChannel()
         {
-            return Connection.Url?.StartsWith("https:", StringComparison.OrdinalIgnoreCase) == true;
+            foreach (var url in Connection.TopologyDiscoveryUrls)
```

Comment:
This should be inverted. The logic is that if any of them uses `http`, then it is false. Otherwise, true.